common/src/main/java/uk/nhs/adaptors/common/util/CodeableConceptUtils.java

```diff
@@ -29,6 +30,16 @@ public static CodeableConcept createCodeableConcept(String code, String system,
         return codeableConcept;
     }
 
+    public static CodeableConcept createCodeableConceptWithDoubleCoding(String code, String system, String display,
+                                                                        String text, String gp2gpSpecificCode) {
+        var codeableConcept = new CodeableConcept().setText(text);
+        var coding = new Coding(system, code, display);
+        var codingWithGP2GPSpecificCode = new Coding("2.16.840.1.113883.2.1.3.2.4.17.101", gp2gpSpecificCode, display);
```

Comment:
Maybe pull the oid out into a const rather than hard coding in the method, something like ACKNOWLEDGEMENT_CODE_SYSTEM